src/styles/common/textStyles.js

```diff
@@ -34,6 +35,14 @@ const rawStyles = {
     fontFamily: THEME.FONTS.DEFAULT,
     fontSize: THEME.rem(1),
     textAlign: 'center'
+  },
+
+  linkText: {
+    width: '100%',
+    fontSize: scale(18),
```

Comment:
We are trying to get rid of `scale`. The new replacement is `THEME.rem(x)`. The idea is that all layout should be relative to the height of a line of text, which is 1 rem. So, the `fontSize` here should be 1 rem, or `THEME.rem(1)`.